Code/Editor/EditorViewportWidget.cpp

```diff
@@ -2803,11 +2507,28 @@ void EditorViewportWidget::SetViewFromEntityPerspective(const AZ::EntityId& enti
     SetViewAndMovementLockFromEntityPerspective(entityId, false);
 }
 
-void EditorViewportWidget::SetViewAndMovementLockFromEntityPerspective(const AZ::EntityId& entityId, bool lockCameraMovement)
+void EditorViewportWidget::SetViewAndMovementLockFromEntityPerspective(const AZ::EntityId& entityId, [[maybe_unused]] bool lockCameraMovement)
 {
-    if (!m_ignoreSetViewFromEntityPerspective)
+    // This is an editor event, so is only serviced during edit mode, not play game mode
+    // 
+    if (m_playInEditorState != PlayInEditorState::Editor)
+    {
+        AZ_Warning("EditorViewportWidget", false,
+            "Tried to change the editor camera during play game in editor; this is currently unsupported"
+        );
+        return;
+    }
+
+    AZ_Assert(lockCameraMovement == false, "SetViewAndMovementLockFromEntityPerspective with lockCameraMovement == true not supported");
+
+    if (entityId.IsValid())
+    {
+        EBUS_EVENT_ID(entityId, Camera::CameraRequestBus, MakeActiveView);
```

Comment:
nit: Could the please use the more modern EBus calling syntax please

I think it should just be:

`Camera::CameraRequestBus::Event(entityId, &Camera::CameraRequestBus::Events::MakeActiveView);`